doc/developers/contributing.rst

```diff
@@ -1253,6 +1253,40 @@ from high-level questions to a more detailed check-list.
 
 :ref:`saved_replies` includes some frequent comments that reviewers may make.
 
+.. _communication:
+
+Communication Guidelines
+------------------------
+
+Reviewing open pull requests (PRs) helps move the project forward. It is a
+great way to get familiar with the codebase and should motivate the
+contributor to keep involved in the project. [1]_
+
+- Every PR, good or bad, is an act of generosity. Opening with a positive
+  comment will help the author feel rewarded, and your subsequent remarks may
+  be heard more clearly. You may feel good also.
+- Begin if possible with the large issues, so the author knows they’ve been
+  understood. Resist the temptation to immediately go line by line, or to open
+  with small pervasive issues.
+- Do not let perfect be the enemy of the good. If you find yourself making
+  many small suggestions that are a matter of subjective taste rather than
+  somewhat objective, consider that the frontier between a subjective nit and
+  an objective improvement is often subjective. The following approaches are
```

Comment:
This new version doesn't have the same intended meaning as the original one I suggested in  https://github.com/scikit-learn/scikit-learn/pull/20048#issuecomment-833418103:

> > Regarding docs: typos, grammar issues and disambiguations are better addressed immediately.

> I have skipped this sentence, as this is not about communication but about review checklist.

Then I would suggest to also remove "the frontier between a subjective nit and
  an objective improvement is often subjective" because they go hand in hand.

Reply:
BTW:

> this is not about communication but about review checklist

This is true, and in fact this entire item is about review checklist. So following that logic, we can remove it altogether.

Reply:
This item is meant to be about "how to formulate the review".
The grammar and typo thing is about "what to look for in a review".